main.go

```diff
@@ -57,7 +57,7 @@ func main() {
 	// handling for saved objects instead of a full savegame
 	if *objin != "" {
 		objs = file.NewJSONOps(filepath.Dir(*objin))
-		objdir = file.NewDirOps(filepath.Dir(*objin))
+		objdir = file.NewDirOps(filepath.Dir(*objout))
```

Comment:
Let's break this into it's own change. I'm not sure that it is a bug. And if it is, then that means the whole feature is not well tested

Reply:
Without this change, there will be empty folders created in the folder of the `objin` file that mimic the actual folder structure of the output.